### PR TITLE
fix(Editable): editable style

### PR DESCRIPTION
**Pull request by suhongyan**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?**

**Please check if the PR fulfills these requirements**

*   [ ] Follow our contributing docs
*   [ ] Docs have been added/updated
*   [ ] Tests have been added; existing tests pass

**Other information**:

### CODE REVIEW

**Comment by PannanaAlex** [comment]
lgtm

**Comment by maryyang1234** [comment]
lgtm

File: src/components/EditableTable/style/index.js [hunk 1]
```diff
@@ -60,9 +60,11 @@ export const AddBar = withProps()(
             text-align: center;
             line-height: 60px;
             border-top: 1px solid ${DT.T_COLOR_LINE_DEFAULT_DARK};
-
+            .uc-fe-button {
+                border-radius: 4px;
```
Comment (ucloud-pccl) [inline src/components/EditableTable/style/index.js:64]: 注意用designToken

File: src/components/EditableTable/style/index.js [hunk 2]
```diff
@@ -78,9 +80,11 @@ export const AddBar = withProps()(
                     background: ${DT.T_COLOR_BG_DISABLED_LIGHT};
                     color: ${DT.T_COLOR_TEXT_DISABLED};
                     cursor: default;
+                    background: #eef3ff;
+                    border-radius: 4px;
```
Comment (ucloud-pccl) [inline src/components/EditableTable/style/index.js:84]: 注意用designToken

**Review by ucloud-pccl — COMMENTED** [review]
注意用designToken

**Comment by suhongyan** [comment]
done

**Pull request merged**